DatadogCore/Tests/DatadogObjc/DDRUMConfigurationTests.swift

```diff
@@ -110,7 +110,7 @@ class DDRUMConfigurationTests: XCTestCase {
         let random: TimeInterval = .mockRandom()
         objc.appHangThreshold = random
         XCTAssertEqual(objc.appHangThreshold, random)
-        XCTAssertEqual(swift.appHangThreshold, random)
+        XCTAssertEqual(swift.appHangThreshold, random == 0 ? nil : random)
```

Comment:
I feel like this test modification is trying to accommodate the changes in logic, rather than ensuring it accurately validates the new logic.

We could add a new test specifically for the `0` value. And to avoid avoid `mockRandom` returning `0` in the original test, making it fail, we could use another extension, such as the `mockRandom` `Double extension` in `FoundationMocks`, which takes parameters.

Reply:
yes, it can be see as an attempt to cut the corner.

I've pushed an update: one test will test strictly positive value (I didn't introduce additional extension method, but rather re-used existing one providing explicit args), another will test 0 value to disable app hangs monitoring.

Let me know what you think.

Reply:
It looks great! 👌✅